docs-src/http-api.md

```diff
@@ -0,0 +1,530 @@
+          "protected": "eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDprZXk6ejZNa2ZaNlM0TlZWVEV1dHM4bzV4RnpSTVI4ZUM2WTFibmdvQlFOblhpQ3ZoSDhII3o2TWtmWjZTNE5WVlRFdXRzOG81eEZ6Uk1SOGVDNlkxYm5nb0JRTm5YaUN2aEg4SCJ9"
+        }
+      ],
+      "link": "bafyreigzxkki35e3pd4r6zvowmp25fly7yx5bjqwkojwiftpe5r3xx4gnu"
+    },
+    "linkedBlock": "pGJpZNgqWCYAAYUBEiDRQJ7VCtGQWcLlmFpitGoSP35ntX7fKJeFWJ8zKz2+Z2RkYXRhgaNib3BjYWRkZHBhdGhlL21vcmVldmFsdWUY6mRwcmV22CpYJgABhQESINFAntUK0ZBZwuWYWmK0ahI/fme1ft8ol4VYnzMrPb5nZmhlYWRlcqFrY29udHJvbGxlcnOA"
+  }
+}' -H "Content-Type: application/json"
+
+{
+  "docId": "kjzl6cwe1jw14ahmwunhk9yjwawac12tb52j1uj3b9a57eohmhycec8778p3syv",
+  "state": {
+    "doctype": "tile",
+    "content": {
+      "title": "My first Document"
+    },
+    "metadata": {
+      "schema": null,
+      "controllers": [
+        "did:key:z6MkfZ6S4NVVTEuts8o5xFzRMR8eC6Y1bngoBQNnXiCvhH8H"
+      ]
+    },
+    "signature": 2,
+    "anchorStatus": "PENDING",
+    "log": [
+      {
+        "cid": "bagcqcera2faj5vik2giftqxftbngfndkci7x4z5vp3psrf4flcptgkz5xztq",
+        "type": 0
+      },
+      {
+        "cid": "bagcqcera3fkje7je4lvctkam4fvi675avtcuqgrv7dn6aoqljd5lebpl7rfq",
+        "type": 1
+      }
+    ],
+    "anchorScheduledFor": "12/30/2020, 1:15:00 PM",
+    "next": {
+      "content": {
+        "title": "My first Document",
+        "more": 234
+      },
+      "metadata": {
+        "schema": null,
+        "controllers": []
+      }
+    }
+  }
+}
+
+
+```
+
+
+
+# Pins
+
+The pins api endpoint can be used to manipulate the pin set. The pin set is all of the documents that a node maintains the state of. Any document opened by the node that is not pinned will eventually be garbage collected from the node.
+
+## Add a document to the pin set
+This method adds the document with the given [[DocID]] to the pin set. 
+
+*`⚠️ This method is not available if gateway mode is enabled.`*
+
+### Request
+`POST /api/v0/pins/:docid`
+
+Here, `:docid` should be replaced by the string representation of the DocID of the document that is being requested.
+
+### Response
+
+* `docId` - the [[DocID]] of the document which was pinned, string
+* `isPinned` - whether the document was pinned, boolean - `true`
+
+### Example
+
+```bash
+curl http://localhost:7007/api/v0/pins/k2t6wyfsu4pg2qvoorchoj23e8hf3eiis4w7bucllxkmlk91sjgluuag5syphl -X POST
+
+{
+  "docId": "k2t6wyfsu4pg2qvoorchoj23e8hf3eiis4w7bucllxkmlk91sjgluuag5syphl",
+  "isPinned": true
+}
+```
+
+## Remove a document from the pin set
+This method removes the document with the given [[DocID]] from the pin set.
+
+*`⚠️ This method is not available if gateway mode is enabled.`*
+
+### Request
+`DELETE /api/v0/pins/:docid`
+
+Here, `:docid` should be replaced by the string representation of the DocID of the document that is being requested.
+
+### Response
+
+* `docId` - the [[DocID]] of the document which was unpinned, string
+* `isPinned` - whether the document was pinned, boolean - `false`
```

Comment:
this doesn't actually return whether the document was pinned previously.  It is just always `false` if the unpin command was successful